lib/mix/tasks/generate_resource_diagrams.ex

```diff
@@ -31,6 +31,8 @@ defmodule Mix.Tasks.Ash.GenerateResourceDiagrams do
         Enum.map(List.wrap(opts[:only]), &Path.expand/1)
       end
 
+    format = Keyword.get(opts, :format, "plain")
```

Comment:
We aren't parsing the format in this case yet.

Reply:
Good catch. Fixed